e2etest/zt_newe2e_file_test.go

```diff
@@ -549,8 +550,11 @@ func (s *FileTestSuite) Scenario_CopyTrailingDotUnsafeDestination(svm *ScenarioV
 
 // Test:
 // - correct number of non-empty files are uploaded when file share quota is hit
-// - TODO 10.29.0 release: transfers complete successfully with resume command
 func (s *FileTestSuite) Scenario_UploadFilesWithQuota(svm *ScenarioVariationManager) {
+	if svm.Dryrun() {
```

Comment:
This will cause the scenario to generate no variations and effectively not get ran.